lib/phctl.rb

```diff
@@ -20,7 +20,7 @@ def run_job(klass, *args, **kwargs)
 
       def run_common_job(klass, options, *args, **kwargs)
```

Comment:
You renamed `*args` and `**kwargs` in 2 other places in this PR, but not here. 
I cannot directly sense intention, but it _feels_ like you missed one.

Reply:
Might be a standardrb issue; I'll take a look

Reply:
Yes, I think standardrb kept it in this case because the anonymous use of `*` and `**` only appears to work if they are the individual last argument in a method call. I wasn't familiar with this use and don't particularly love it, but I'm not going to argue with standardrb...